testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -12,10 +12,6 @@
 # * if there is no private network ($private_net is nil)
 # * if there is no PXE boot minion ($pxeboot_mac is nil)
 
-# TODO: This feature needs to be refactored for RBS containerized,
-#       as it does not support Salt formulas.
-#       Additionally, we have this bug that impedes to use mgr-bootstrap command https://bugzilla.suse.com/show_bug.cgi?id=1220864
-
 @skip_if_containerized_server
```

Comment:
I would assume we also need to split this feature in two, one for traditional and another for containerized. Make sense?

Reply:
That is a very good and important question.

My first tests seem to show I can cope with one single code for both. But I'm not advanced enough, so yes, we might need to duplicate it :cry:  .

Reply:
I doubt we can make it with one, the reason is that for a traditional RBS, the doc explains that we need to use its Salt Formulas to configure some stuff, while in a containerized RBS the doc tells you to handle it differently.

Reply:
Naturally, the tag `@skip_if_containerized_server` at feature level will go away in both hypothesis.

Reply:
I am not that sure about the formulas: in my understanding, we drop the DHCP and DNS formulas, but we keep the other ones.

But I might be wrong here. I lack experience.